src/main/twirl/uk/gov/hmrc/play/views/helpers/singleCheckbox.scala.html

```diff
@@ -19,13 +19,13 @@
 @import views.html.helper._
 
 @elements = @{ new FieldElements(field.id, field, null, args.toMap, messages) }
-@value = @{ field.value match { case Some(x) => x case None => "false" case x => x }}
-<div class="form-field-single @if(elements.hasErrors) {error}">
+@value = @{ field.value match { case Some(x) => x case None => "false" case x => x } }
+@style = @{ elements.args.get('_inLine) match { case Some(true) => "form-checkbox" case _ => "block-label" } }
+
+<label class="form-field-single @if(elements.args.get('_labelClass)){ @elements.args.get('_labelClass) } @style @if(elements.hasErrors){ error form-field--error } @if(value=="true" && style=="block-label"){ selected }" for="@elements.id">
```

Comment:
Instead of `@if(value=="true" && style=="block-label"){ selected }` you can just do `@field.value.map(selected)` (as it is an `Option`) I'm not quite sure why an `Option` has been converted to a boolean in this way